File: app/src/main/java/com/ladsoft/bakingapp/data/network/entity/StepPayload.java
package com.ladsoft.bakingapp.data.network.entity;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class StepPayload {

    @Expose @SerializedName("id")
    private long id;

    @Expose @SerializedName("shortDescription")
    private String shortDescription;

    @Expose @SerializedName("description")
    private String description;

    @Expose @SerializedName("videoURL")
    private String videoUrl;

    @Expose @SerializedName("thumbnailURL")
    private String thumbnailUrl;

    public long getId() {
        return id;
    }

    public String getShortDescription() {
        return shortDescription;
    }

    public String getDescription() {
        return description;
    }

    public String getVideoUrl() {
        return videoUrl;
    }

    public String getThumbnailUrl() {
        return thumbnailUrl;
    }
}

File: app/src/main/java/com/ladsoft/bakingapp/data/network/entity/StepPayload.kt
package com.ladsoft.bakingapp.data.network.entity


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class StepPayload (
    @Expose
    @SerializedName("id")
    val id: Long,

    @Expose
    @SerializedName("shortDescription")
    val shortDescription: String?,

    @Expose
    @SerializedName("description")
    val description: String?,

    @Expose
    @SerializedName("videoURL")
    val videoUrl: String?,

    @Expose
    @SerializedName("thumbnailURL")
    val thumbnailUrl: String?
)
